.gitlab-ci.yml

```diff
@@ -30,10 +30,27 @@ stages:
     reports:
       junit: build/junit.xml
 
+####
+# Template
+.run_project_integration_tests:
+  script:
+    #Use pre-existing allocation if any
+    - JOBID=$(if [[ $SYS_TYPE = toss_* ]]; then squeue -h --name=${PROJECT_ALLOC_NAME} --format=%A; fi)
+    - ASSIGN_ID=$(if [[ -n "${JOBID}" ]]; then echo "--jobid=${JOBID}"; fi)
+   #BUILD + TEST
+    - echo -e "section_start:$(date +%s):run_project_integration_tests\r\e[0K
+      Run project integration tests ${CI_PROJECT_NAME}"
+    - ${ALLOC_COMMAND} .gitlab/run_project_integration_tests.sh
+    - echo -e "section_end:$(date +%s):build_and_test\r\e[0K"
+  artifacts:
+    reports:
+      junit: build/junit.xml
+
 
 # This is where jobs are included
 include:
   - local: .gitlab/build_lassen.yml
   - local: .gitlab/build_quartz.yml
 # white238: Turned off while tioga is down for maintenance and is fickle
```

Comment:
We should be able to turn this back on now.. Tioga has been behaving for a while now.